packages/core/saved-objects/core-saved-objects-migration-server-internal/src/document_migrator/model_version.ts

```diff
@@ -32,45 +34,54 @@ export const getModelVersionTransforms = ({
       ? typeDefinition.modelVersions()
       : typeDefinition.modelVersions ?? {};
 
-  const transforms = Object.entries(modelVersionMap)
-    .filter(([_, definition]) => !!definition.modelChange.transformation)
-    .map<Transform>(([rawModelVersion, definition]) => {
-      const modelVersion = assertValidModelVersion(rawModelVersion);
-      const virtualVersion = modelVersionToVirtualVersion(modelVersion);
-      return {
-        version: virtualVersion,
-        transform: convertModelVersionTransformFn({
-          log,
-          modelVersion,
-          virtualVersion,
-          definition,
-        }),
-        transformType: TransformType.Migrate,
-      };
-    });
-
-  return transforms;
+  return Object.entries(modelVersionMap).map<Transform>(([rawModelVersion, definition]) => {
+    const modelVersion = assertValidModelVersion(rawModelVersion);
+    const virtualVersion = modelVersionToVirtualVersion(modelVersion);
```

Comment:
To avoid breaking change in the `typeMigrationVersion` field, we're considering that model versions are just version `10`. E.g MV 1 = version `10.1.0`.